### DISCUSSION TITLE
How to make these modification if possible

### DISCUSSION THREAD

**Discussion by jtipt** [tomasklaen/uosc#908] [Q&A] [answered]
Is there a way these modifications can be done? If possible. Thanks
1. Make the entire OSD visible when the mouse is moved, similar to the default behavior of the mpv seek bar, which appears when the mouse is moved and disappears after a few seconds. Currently, with uosc's default settings, the bottom seek bar only shows up when the mouse is moved to the bottom edge of the screen. I would like the bottom bar, as well as other OSD elements, to show up with any mouse movement.
2. Make the full seek bar show up when using mouse-wheel to seek, by default only a thin bar shows up which is hardly noticeable.

**Comment by Sneakpeakcss** [comment, marked as answer]
1. This should be possible with `uosc` config options: `proximity_in=9999` (maybe even higher), `autohide=yes`
2. Do you mean seeking while hovering over the timeline ? Because at that point it should be fully expanded rather than using the 'thin always visible' one. If it's about custom keybinds then you can use [flash-ui](https://github.com/tomasklaen/uosc#flash-ui):

```
Wheel_Up    seek  5 exact; script-binding uosc/flash-timeline
Wheel_Down  seek -5 exact; script-binding uosc/flash-timeline
```

**Reply by jtipt** [reply]
Thanks thats exactly what I was looking for
